src/site/content/en/blog/web-platform-05-2023/index.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title: "New to the web platform in May"
+subhead: >
+  Discover some of the interesting features that landed in stable and beta web browsers during May 2023.
+description: >
+  Discover some of the interesting features that landed in stable and beta web browsers during May 2023.
+date: 2023-05-31
+hero: image/kheDArv5csY6rvQUJDbWRscckLr1/ozivqeizYMb6e6KTwiob.jpg
+alt: A selection of colored pencils.
+authors:
+  - rachelandrew
+tags:
+  - blog
+  - new-to-the-web
+---
+
+## Stable browser releases
+
+In May 2023, [Firefox 113](https://developer.mozilla.org/docs/Mozilla/Firefox/Releases/113), [Chrome 113](https://developer.chrome.com/blog/new-in-chrome-113/), [Chrome 114](https://developer.chrome.com/blog/new-in-chrome-114/), and [Safari 16.5](https://developer.apple.com/documentation/safari-release-notes/safari-16_5-release-notes) became stable. Let's take a look at what this means for the web platform.
+
+{% Aside 'caution' %}
+We include browser compatibility data pulled from MDN in these posts which may not have updated for very recent browser releases. The post will show the correct information as soon as it appears in the [browser-compat-data release](https://github.com/mdn/browser-compat-data/releases).
+{% endAside %}
+
+### WebGPU
+
+Chrome 113 includes [WebGPU](https://developer.chrome.com/blog/webgpu-release/), the successor to the WebGL and WebGL 2 graphics APIs for the Web. It provides modern features such as GPU compute, lower overhead access to GPU hardware, the ability to render to multiple canvases from a single graphics device, and better, more predictable performance.
+
+{% BrowserCompat 'api.GPU' %}
+
+### First-Party Sets
+
+[First Party Sets](https://developer.chrome.com/docs/privacy-sandbox/first-party-sets/) is part of the [Privacy Sandbox](https://privacysandbox.com/). It is a way for organizations to declare relationships among sites, so that browsers allow limited third-party cookie access for specific purposes. This began a staged rollout in Chrome 113. 
```

Comment:
```suggestion
[First-Party Sets (FPS)](https://developer.chrome.com/docs/privacy-sandbox/first-party-sets/) is part of the [Privacy Sandbox](https://privacysandbox.com/). It is a way for organizations to declare relationships among sites, so that browsers can decide when to allow limited third-party cookie access for sites within a set. FPS began a staged rollout in Chrome 113. 
```